### PR TITLE
Ncl 1116

### PR DESCRIPTION
**Pull request by pgier**
Adding new product milestone detail page which includes distributed artifacts and performed builds

### CODE REVIEW

File: rest/src/main/webconfig/auth/keycloak.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "realm": "pncredhat",
   "realm-public-key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAjG+B9bN0bQNDzpazVrpIG5SLEGmz7iehC/IjKLkFhwetP+nIVQi5hn59osJNrQ/vTGDdEYWLProLzkhJHRQzwZs2ucvyolaL/FcQ+l7UCxrewyhSsIubra14dJwhglKqQS+ZD2nRcQuiMJ97J9D9gqnk8z31mUO1Oq57HSIEwq/VH8YEWsOSnvd1ZDoYiKffnrghq1DTaEgPZm3aKPMmPkAS3n7ClC2Doww8r18EOEh+y8cea0M3fOpFL81137likn1m0phSHmdJ4MLVhUwgFn50UDGTjzB3pFAkUWE7s473C4zxAamLA1740hnUgXoCL36KKaMbZLQ4ZJnQEZQL6wIDAQAB",
-  "auth-server-url": "@url-of-your-keycloak-server@",
+  "auth-server-url": "https://ncl-keycloak.stage.engineering.redhat.com/auth",
```
Comment (alexcreasy) [inline rest/src/main/webconfig/auth/keycloak.json]: Accidental check in here! We could really do with a better way of injecting authentication configuration into the build.

**Comment by thescouser89** [comment]
There are some unused variables added. check the logs for jshint

**Comment by thescouser89** [comment]
retest this please

**Comment by pgier** [comment]
ok, the keycloak url and the unused variables should be fixed now, thanks for the feedback!

**Comment by jbartece** [comment]
retest this please

File: ui/app/milestone/milestone-controllers.js [hunk 2]
```diff
@@ -21,6 +21,27 @@
...
+    'distributedArtifacts',
+    'performedBuilds',
+    function($scope, $state, $stateParams, productDetail, versionDetail, milestoneDetail,
+        distributedArtifacts, performedBuilds) {
+
+      var that = this;
+      that.product = productDetail;
+      that.productVersion = versionDetail;
+      that.milestone = milestoneDetail;
+      that.distributedArtifacts = distributedArtifacts.data;
```
Comment (matedo1) [inline ui/app/milestone/milestone-controllers.js]: Hi @pgier ,

http://localhost:9000/#/product/1/version/1/milestone/1/detail Shipped Artifacts part is empty until page is refreshed.

I think there data attribute is not resolved properly.

**Comment by matedo1** [comment]
I'm closing this issue, new updated PR is raised here https://github.com/project-ncl/pnc/pull/1116

**Pull request closed without merging**